Source/CombatExtended/CombatExtended/AmmoGeneralizer.cs

```diff
@@ -84,6 +84,28 @@ static AmmoGeneralizer()
                         
                     }
                 }
+
+                var toFixComps = DefDatabase<ThingDef>.AllDefs.Where(x => x.comps?.Any(x => x is CompProperties_Reloadable) ?? false);
+
+                foreach (var def in toFixComps)
+                {
+                    
+                    var compProps = def.GetCompProperties<CompProperties_Reloadable>();
+                    if (compProps.ammoDef is AmmoDef am)
+                    {
+                        Log.Message(def.label);

```

Comment:
Some reason to keep this, without any annotation, in the release version?

Reply:
simply forgot to